packages/components/src/MessageBar/MessageBar.tsx

```diff
@@ -239,6 +240,24 @@ const MessageBarContent = styled.div`
   padding: 0 ${({ theme: { space } }) => space.large};
 `
 
+const backgroundColor = variant({
+  prop: 'intent',
+  variants: {
+    critical: {
+      backgroundColor: 'criticalAccent',
+    },
+    inform: {
+      backgroundColor: 'informAccent',
+    },
+    positive: {
+      backgroundColor: 'positiveAccent',
+    },
+    warn: {
+      backgroundColor: 'warnAccent',
+    },
+  },
+})
```

Comment:
I've been tending towards using `background` over `background-color`. No huge argument for it besides that it's slightly more terse and if other CSS specifices `background` it can be confusing to have interactions between `background` and `backgroundColor`

```suggestion
const background = variant({
  prop: 'intent',
  variants: {
    critical: {
      background: 'criticalAccent',
    },
    inform: {
      background: 'informAccent',
    },
    positive: {
      background: 'positiveAccent',
    },
    warn: {
      background: 'warnAccent',
    },
  },
})
```

Reply:
I originally tried background, but the color does not get applied.  The theme values name is set as the value of `background`. Any thoughts?
![image](https://user-images.githubusercontent.com/170681/108072514-d8e13d80-701b-11eb-892a-3828da9b26c2.png)

Reply:
Oh - weird, I forgot that `variant` is doing `styled-system` magic here. In testing it turns out that `bg` works too but also outputs `backgroundColor` so 🤷